app/code/Magento/CatalogImportExport/Model/Import/Product/CategoryProcessor.php

```diff
@@ -204,4 +221,15 @@ public function getCategoryById($categoryId)
     {
         return isset($this->categoriesCache[$categoryId]) ? $this->categoriesCache[$categoryId] : null;
     }
+
+    /**
+     * Standardize a string by lowercase-ing it
+     *
+     * @param string $string
+     * @return string
+     */
+    protected function standardizeString($string)
```

Comment:
Yes but if in the future we have to change it with more complex checks, we just have to change the function's code instead of changing all the pieces of code where appears `mb_strtolower`. It's more efficient i think.